### PR TITLE
Tera Shards Icons

### PR DESCRIPTION
**Pull request by AlexOn1ine**
Tera Shard icons. Supersedes #3080.

Full credit to PacFire.

https://github.com/rh-hideout/pokeemerald-expansion/assets/93446519/482769e3-ce5f-4f56-9cb2-c69670cc17b7

### CODE REVIEW

File: src/data/graphics/items.h [hunk 1]
```diff
@@ -1988,7 +1988,63 @@ const u32 gItemIconPalette_TeraOrb[] = INCBIN_U32("graphics/items/icon_palettes/
 const u32 gItemIcon_TinyBambooShoot[] = INCBIN_U32("graphics/items/icons/tiny_bamboo_shoot.4bpp.lz");
 const u32 gItemIconPalette_TinyBambooShoot[] = INCBIN_U32("graphics/items/icon_palettes/tiny_bamboo_shoot.gbapal.lz");
 
-// Tera Shards here
+// Tera Shards
+
+const u32 gItemIcon_NormalTeraShard[] = INCBIN_U32("graphics/items/icons/normal_tera_shard.4bpp.lz");
+const u32 gItemIconPalette_NormalTeraShard[] = INCBIN_U32("graphics/items/icon_palettes/normal_tera_shard.gbapal.lz");
+
+const u32 gItemIcon_FireTeraShard[] = INCBIN_U32("graphics/items/icons/normal_tera_shard.4bpp.lz");
```
Comment (AsparagusEduardo) [inline src/data/graphics/items.h:1996]: Remove the repeate icons.

File: src/data/graphics/items.h [hunk 2]
```diff
@@ -1988,7 +1988,63 @@ const u32 gItemIconPalette_TeraOrb[] = INCBIN_U32("graphics/items/icon_palettes/
 const u32 gItemIcon_TinyBambooShoot[] = INCBIN_U32("graphics/items/icons/tiny_bamboo_shoot.4bpp.lz");
 const u32 gItemIconPalette_TinyBambooShoot[] = INCBIN_U32("graphics/items/icon_palettes/tiny_bamboo_shoot.gbapal.lz");
 
-// Tera Shards here
+// Tera Shards
+
+const u32 gItemIcon_NormalTeraShard[] = INCBIN_U32("graphics/items/icons/normal_tera_shard.4bpp.lz");
```
Comment (AsparagusEduardo) [inline src/data/graphics/items.h:1993]:
```suggestion
const u32 gItemIcon_TeraShard[] = INCBIN_U32("graphics/items/icons/tera_shard.4bpp.lz");
```

**Review by AsparagusEduardo — CHANGES_REQUESTED** [review]

File: src/data/graphics/items.h (hunk 1 above)
Comment (AlexOn1ine) [reply src/data/graphics/items.h]: suggestion applied

File: src/data/graphics/items.h (hunk 2 above)
Comment (AlexOn1ine) [reply src/data/graphics/items.h]: applied

**Review by AsparagusEduardo — APPROVED** [review]

**Pull request merged**